### ISSUE TITLE
`.postcssrc.js` requires `autoprefixer` but it isn't listed as a dependency.

### ISSUE THREAD

**Issue by Naryosha** [quasarframework/quasar-starter-kit#89]
**Describe the bug**
`.postcssrc.js` requires `autoprefixer` but it isn't listed as a dependency in the template [`package.json`](https://github.com/quasarframework/quasar-starter-kit/blob/c1b7dfe00870f739430ac46f93e3d7dbf799f80a/template/package.json)

**To Reproduce**
1. `npm i -g pnpm`
2. `pnpm i -g @quasar/cli`
3. `quasar create quasar-app`

```
? Project name (internal usage for dev) quasar-app
? Project product name (must start with letter if building mobile apps) Quasar App
? Project description A Quasar Framework app
? Pick your favorite CSS preprocessor: (can be changed later) SCSS
? Pick a Quasar components & directives import strategy: (can be changed later) Auto import
? Check the features needed for your project: ESLint, Vuex, Axios, Vue-i18n
? Pick an ESLint preset: Airbnb
? Cordova/Capacitor id (disregard if not building mobile apps) com.noshyaar.quasarapp
? Should we run `npm install` for you after the project has been created? (recommended) no
```

4. `cd quasar-app`
5. `pnpm i`
6. `quasar dev`

**Expected behavior**
Running development server

**Screenshots**
\-

**Platform (please complete the following information):**
OS: Ubuntu 19.10 eoan
Node: v13.0.1
NPM: 6.12.0
Yarn: -
pnpm: 4.9.3
Browsers: Mozilla Firefox 72.0.2
iOS: -
Android: -
Electron: -

**Additional context**
```
$ quasar dev

 Dev mode.......... spa
 Pkg quasar........ v1.8.5
 Pkg @quasar/app... v1.5.3
 Debugging......... enabled

 app:quasar-conf Reading quasar.conf.js +0ms
 app:dev Checking listening address availability (0.0.0.0:8080)... +5ms
(node:10728) UnhandledPromiseRejectionWarning:   Error: Cannot find module 'autoprefixer'
  Require stack:
  - /home/noshyaar/git/quasar-app/.postcssrc.js
  - /home/noshyaar/git/quasar-app/node_modules/.pnpm/registry.npmjs.org/@quasar/app/1.5.3/node_modules/@quasar/app/lib/webpack/inject.style-rules.js
  - /home/noshyaar/git/quasar-app/node_modules/.pnpm/registry.npmjs.org/@quasar/app/1.5.3/node_modules/@quasar/app/lib/webpack/create-chain.js
  - /home/noshyaar/git/quasar-app/node_modules/.pnpm/registry.npmjs.org/@quasar/app/1.5.3/node_modules/@quasar/app/lib/webpack/index.js
  - /home/noshyaar/git/quasar-app/node_modules/.pnpm/registry.npmjs.org/@quasar/app/1.5.3/node_modules/@quasar/app/lib/quasar-config.js
  - /home/noshyaar/git/quasar-app/node_modules/.pnpm/registry.npmjs.org/@quasar/app/1.5.3/node_modules/@quasar/app/bin/quasar-dev
  - /home/noshyaar/git/quasar-app/node_modules/.pnpm/registry.npmjs.org/@quasar/app/1.5.3/node_modules/@quasar/app/bin/quasar
  - /home/noshyaar/.nvm/versions/node/v13.0.1/pnpm-global/3/node_modules/.pnpm/registry.npmjs.org/@quasar/cli/1.0.5/node_modules/@quasar/cli/bin/quasar
```

**Comment by rstoenescu** [comment]
Hi,

"autoprefixer" is supplied by "@quasar/app". Highly recommending (as in the docs) to use a better package manager: yarn, for a headache-free experience.

**Comment by zkochan** [comment]
I am not sure what a "better package manager" means. If this issue is because a non-declared dependency is used, then this will also fail with Yarn v2.

**Comment by rstoenescu** [comment]
Maybe my wording was "off". So here's another try.

You can also install the same "autoprefixer" version in your project folder and still use pnpm. But if you install packages supplied by quasar/app yourself, you are essentially throwing away all the stability that q/app offers out of the box. We thoroughly test packages and pin them to "safe" versions (and also upgrade them really fast) so that the developers can focus on their work rather than the tooling. We've had so many incidents where certain versions of some packages broke stuff. Our philosophy is to make everything easier for you, not harder.

So aside from the possibility to make pnpm play nicely with Quasar projects (available today), my personal opinion, and everyone can feel free to disagree with me, is that a package manager should not should not matter so much when compared to the quality of the UI that you can make with a framework (including exceptional out of the box support for all platforms, be them desktop, mobile or hybrid) and the overall experience that you get from it -- the ability to focus on your end-result which is the only thing that your customers will care about.

Bottom line, let's make great apps and not build walls against each other.

**Comment by RaviH** [comment]
@rstoenescu quasar is amazing. Period. `pnpm` support, while not important, would be nice to have. 

I get: 

`~/todo-app/.quasar/client-entry.js in multi ./node_modules/.pnpm/webpack-dev-server@3.11.0_webpack@4.44.1/node_modules/webpack-dev-server/client?http://0.0.0.0:8080 (webpack)/hot/dev-server.js ./.quasar/client-entry.js`

**Comment by moghthalkushal** [comment]
Yarn or Npm the issue persists
`
![image](https://user-images.githubusercontent.com/5137212/113253679-ee2ed480-9308-11eb-8f3a-877e20164243.png)
`

**Comment by moghthalkushal** [comment]
The solution works while quasar create.

But, when I check in the code and pull the code as a fresh copy into new folder and perform yarn or npm it fails.

client-app % yarn
yarn install v1.22.10
info No lockfile found.
[1/5] 🔍  Validating package.json...
[2/5] 🔍  Resolving packages...
warning @quasar/app > webpack-dev-server > chokidar@2.1.8: Chokidar 2 will break on node v14+. Upgrade to chokidar 3 with 15x less dependencies.
warning @quasar/app > webpack-dev-server > chokidar > fsevents@1.2.13: fsevents 1 will break on node v14+ and could be using insecure binaries. Upgrade to fsevents 2.
warning @quasar/app > webpack > watchpack > watchpack-chokidar2 > chokidar@2.1.8: Chokidar 2 will break on node v14+. Upgrade to chokidar 3 with 15x less dependencies.
warning @quasar/app > webpack-dev-server > chokidar > braces > snapdragon > source-map-resolve > resolve-url@0.2.1: https://github.com/lydell/resolve-url#deprecated
warning @quasar/app > webpack-dev-server > chokidar > braces > snapdragon > source-map-resolve > urix@0.1.0: Please see https://github.com/lydell/urix#deprecated
warning babel-eslint@10.1.0: babel-eslint is now @babel/eslint-parser. This package will no longer receive updates.
[3/5] 🚚  Fetching packages...
[4/5] 🔗  Linking dependencies...
warning " > vue-i18n@9.0.0" has unmet peer dependency "vue@^3.0.0".
warning "@typescript-eslint/eslint-plugin > tsutils@3.21.0" has unmet peer dependency "typescript@>=2.8.0 || >= 3.2.0-dev || >= 3.3.0-dev || >= 3.4.0-dev || >= 3.5.0-dev || >= 3.6.0-dev || >= 3.6.0-beta || >= 3.7.0-dev || >= 3.7.0-beta".
[5/5] 🔨  Building fresh packages...
success Saved lockfile.
✨  Done in 41.04s.

client-app % quasar dev

 Dev mode.......... spa
 Pkg quasar........ v2.0.0-beta.11
 Pkg @quasar/app... v3.0.0-beta.10
 Debugging......... enabled

 Configured browser support (at least 89.53% of global marketshare):
 · Chrome for Android >= 89
 · Firefox for Android >= 86
 · QQ for Android >= 10.4
 · UC for Android >= 12.12
 · Android >= 89
 · Baidu >= 7.12
 · Chrome >= 88
 · Edge >= 88
 · Firefox >= 85
 · IE >= 11
 · iOS >= 13.4-13.7
 · Kaios >= 2.5
 · Opera Mini >= all
 · Opera Mobile >= 62
 · Opera >= 72
 · Safari >= 13.1
 · Samsung >= 12.0

 App · Reading quasar.conf.js
 App · Checking listening address availability (0.0.0.0:8080)...
 App · Transpiling JS (Babel active)
(node:11366) UnhandledPromiseRejectionWarning:   Error: Cannot find module '/Users/kushalk/repos/MBE-Quasar/Matchbox/client-app/.postcssrc.js'
  Require stack:
  - /Users/kushalk/repos/MBE-Quasar/Matchbox/client-app/node_modules/@quasar/app/lib/webpack/inject.style-rules.js
  - /Users/kushalk/repos/MBE-Quasar/Matchbox/client-app/node_modules/@quasar/app/lib/webpack/create-chain.js
  - /Users/kushalk/repos/MBE-Quasar/Matchbox/client-app/node_modules/@quasar/app/lib/webpack/index.js
  - /Users/kushalk/repos/MBE-Quasar/Matchbox/client-app/node_modules/@quasar/app/lib/quasar-conf-file.js
  - /Users/kushalk/repos/MBE-Quasar/Matchbox/client-app/node_modules/@quasar/app/bin/quasar-dev
  - /Users/kushalk/repos/MBE-Quasar/Matchbox/client-app/node_modules/@quasar/app/bin/quasar
  - /Users/kushalk/.config/yarn/global/node_modules/@quasar/cli/bin/quasar
  
  - loader.js:880 Function.Module._resolveFilename
    internal/modules/cjs/loader.js:880:15
  
  - loader.js:725 Function.Module._load
    internal/modules/cjs/loader.js:725:27
  
  - loader.js:952 Module.require
    internal/modules/cjs/loader.js:952:19
  
  - helpers.js:88 require
    internal/modules/cjs/helpers.js:88:18
  
  - inject.style-rules.js:7 Object.<anonymous>
    [client-app]/[@quasar]/app/lib/webpack/inject.style-rules.js:7:23
  
  - loader.js:1063 Module._compile
    internal/modules/cjs/loader.js:1063:30
  
  - loader.js:1092 Object.Module._extensions..js
    internal/modules/cjs/loader.js:1092:10
  
  - loader.js:928 Module.load
    internal/modules/cjs/loader.js:928:32
  
  - loader.js:769 Function.Module._load
    internal/modules/cjs/loader.js:769:14
  
  - loader.js:952 Module.require
    internal/modules/cjs/loader.js:952:19
  

(Use `node --trace-warnings ...` to show where the warning was created)
(node:11366) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 1)
(node:11366) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.

**Comment by pinghe** [comment]
https://github.com/quasarframework/quasar/issues/7581

```
We have plans to create a CLI based on Vite instead of Webpack and that will allow us to officially support Yarn 2
```
